core/org.wso2.carbon.utils/src/main/java/org/wso2/carbon/utils/DiagnosticLogUtils.java

```diff
@@ -0,0 +1,51 @@
+package org.wso2.carbon.utils;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import java.time.DateTimeException;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+
+/**
+ * Utility class for diagnostic logging.
+ */
+public class DiagnosticLogUtils {
+
+    private static final Log log = LogFactory.getLog(DiagnosticLogUtils.class);
```

Comment:
```suggestion
    private static final Log LOG = LogFactory.getLog(DiagnosticLogUtils.class);
```
use all capital for final variables